src/main/java/com/redhat/devtools/intellij/tektoncd/tree/MutableTektonModelSynchronizer.java

```diff
@@ -0,0 +1,135 @@
+/*******************************************************************************
```

Comment:
Why is this needed ? Just wondering why we nee this Tekton specific version where almost 90% of the code is duplicated. At least, we should extends the common implementation